SpiceSharp/Sparse/SparseHelper.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace SpiceSharp.Sparse
+{
+    /// <summary>
+    /// Helper methods for Sparse matrices
+    /// </summary>
+    public static class SparseHelper
+    {
+        /// <summary>
+        /// Constants
+        /// </summary>
+        public const double M_LN2 = 0.69314718055994530942;
```

Comment:
@svenboulanger Why do we need this constants?

Reply:
Removed. They were used in uncalled methods.